src/server/api/models/Messages.ts

```diff
@@ -0,0 +1,79 @@
+import {
+  Model,
+  Table,
+  Column,
+  DataType,
+  AutoIncrement,
+  PrimaryKey,
+  AllowNull,
+  CreatedAt,
+  ForeignKey,
+  BelongsTo,
+} from 'sequelize-typescript';
+import Users from './Users';
+import Topics from './Topics';
+import { Nullable } from '../../../client/types';
+
+type Reactions = {
+  like: number[];
+  dislike: number[];
+  laugh: number[];
+  hooray: number[];
+  confused: number[];
+  heart: number[];
+  rocket: number[];
+  eyes: number[];
+};
+
+type MessageAttributes = {
+  id: number;
+  text: string;
+  userId: number;
+  topicId: number;
+  parentId: Nullable<number>;
+  reactions: Reactions;
+  createdAt: string;
+};
+
+@Table
+class Messages extends Model<MessageAttributes> {
+  @AutoIncrement
+  @PrimaryKey
+  @Column(DataType.INTEGER)
+  id!: number;
+
+  @AllowNull(false)
+  @Column(DataType.STRING)
+  text!: string;
+
+  @ForeignKey(() => Users)
+  @AllowNull(false)
+  @Column({
+    type: DataType.INTEGER,
+    field: 'user_id',
+  })
+  userId!: number;
+
+  @ForeignKey(() => Topics)
+  @AllowNull(false)
+  @Column({
+    type: DataType.INTEGER,
+    field: 'topic_id',
+  })
+  topicId!: number;
+
+  @BelongsTo(() => Messages)
+  @Column(DataType.INTEGER)
```

Comment:
```suggestion
  @Column({
    type: DataType.INTEGER,
    field: 'parent_id'
  })
```
Пройдись, пожалуйста, по всем полям во всех таблицах чтобы было соответствие camelCase -> snake_case для JS -> DB

Reply:
Но я же уже прошёлся в коммитах выше. В JS свойства сделал camelCase, в вот этих вот объектах - snake_case. Тут единственное место где пропустил.